src/api/app/controllers/concerns/webui/rescue_handler.rb

```diff
@@ -52,5 +43,55 @@ class MissingParameterError < RuntimeError; end
       logger.debug "#{exception.class.name} #{exception.message} #{exception.backtrace.join('\n')}"
       render file: Rails.root.join('public/404'), status: 404, layout: false, formats: [:html]
     end
+
+    private
+
+    UNAUTHORIZED_MESSAGE = 'Sorry, you are not authorized to perform this action.'.freeze
+    def unauthorized_message(exception)
+      pundit_action = action_for_query(exception.query)
```

Comment:
The `try` was removed: looks like it's not necessary since we are dealing with `Pundit::NotAuthorizedError` data type.